src/core/src/core_logic/PatchAssessor.py

```diff
@@ -84,6 +84,12 @@ def start_assessment(self):
                 self.telemetry_writer.write_event("Security assessment: " + str(sec_packages), Constants.TelemetryEventLevel.Verbose)
                 self.status_handler.set_package_assessment_status(sec_packages, sec_package_versions, "Security")
 
+                if self.package_manager.get_package_manager_setting(Constants.PKG_MGR_SETTING_IDENTITY) == Constants.APT:
```

Comment:
Non-blocking comment: probably bears refactoring to avoid the generic class having to deviate to a specific instantiation's behaviors. It's not always possible cleanly but it's worthwhile to review.